osbs/cli/main.py

```diff
@@ -336,7 +336,9 @@ def cmd_build(args, osbs):
         'platforms': args.platforms,
         'release': args.release,
         'koji_parent_build': args.koji_parent_build,
-        'isolated': args.isolated
+        'isolated': args.isolated,
+        'signing_intent': args.signing_intent,
+        'compose_ids': args.compose_ids
```

Comment:
Nitpick: end this line with a comma so it doesn't need to be modified when adding new items to it.

Reply:
ok